### DISCUSSION TITLE
2 Homematic Repositories - welches für Installation nutzen?

### DISCUSSION THREAD

**Discussion by zapccu** [SukramJ/aiohomematic#1326] [Q&A]
Ich bin gerade dabei, von ioBroker auf Homeassistant umzusteigen. Der größte Brocken ist meine Homematic-Welt mit 2 CCUs und mehr als 70 Geräten.
HACS habe ich installiert und bin nun auf diese beiden Git-Repos gestoßen:

- https://github.com/danielperna84/custom_homematic/tree/main
- https://github.com/danielperna84/hahomematic/tree/main

Welches sollte man für die Homematic Integration verwenden? Oder werden beide synchron gehalten?

**Comment by danielperna84** [comment]
Das hier: [https://github.com/danielperna84/custom_homematic/](https://github.com/danielperna84/custom_homematic/)
